src/planning/include/config/planning_config.hpp

```diff
@@ -68,6 +74,11 @@ struct PlanningConfig {
 
     simulation_.publishing_visualization_msgs_ = params.publishing_visualization_msgs_;
     simulation_.using_simulated_se_ = params.using_simulated_se_;
+
+    velocity_planning_.safety_speed_ = params.minimum_velocity_;
```

Comment:
It's not ideal to have two names for variables with the same purpose, I'd just chose one and go with it